pkg/build/graph/analysis/bc.go

```diff
@@ -185,3 +284,17 @@ func latestBuild(g osgraph.Graph, bc graph.Node) *buildgraph.BuildNode {
 	sort.Sort(sort.Reverse(buildapi.BuildPtrSliceByCreationTimestamp(builds)))
 	return buildNameToNode[builds[0].Name]
 }
+
+// addInputMarker updates the Marker array ptr input parameter with a new marker constructed from the remaining parameters
+func addInputMarker(markers *[]osgraph.Marker, bcNode graph.Node, istNode graph.Node, imageStreamNode *imagegraph.ImageStreamNode, key string, message string, suggestion osgraph.Suggestion) {
```

Comment:
looking at it again, comparing the invocations of the method vs. the contents of the method, along with some variable naming angst, I decided I no longer wanted this helper; losing it and moving the append back to the two spots in the calling method ... part of next push